{{cookiecutter.project_slug}}/tox.ini

```diff
@@ -1,58 +1,21 @@
-[tox]
-envlist = pytest, sphinx
-skipsdist = True
-skip_missing_interpreters = True
-whitelist_externals = python
-
-[testenv]
-basepython = python
-
-[testenv:pytest]
-conda_channels =
-    conda-forge
-    nodefaults
-deps =
-    kaleido
-    -e .  # installs project
-conda_deps =
-    # pytest
-    pytest
-    pytest-cookies
-    pytest-cov
-    pytest-xdist
-
-    # Package dependencies
-    pytask>=0.2
-    pytask-latex>=0.2
-    pyyaml
-    plotly>=5.5.0
-    statsmodels
-    pandas
-    numpy
-
-commands =
-    pytest {posargs}
-
 [doc8]
 ignore = D004
 max-line-length = 88
 
 [flake8]
```

Comment:
I do not want to throw in another tool, but, please, check out https://github.com/charliermarsh/ruff that combines several of our hooks and even automatically fixes them. It is less error-prone than flake8, faster, and fixes stuff instead of complaining.

You find an example configuration at pytask: https://github.com/pytask-dev/pytask/blob/main/pyproject.toml. I am pretty radical in the sense that I accept all lining rules and disable the ones I do not like.

Reply:
Thanks!!! I'll just follow your style, as usual! 

Btw, you would make my life in following your defaults much easier if you added the yamlfix and yamllint hooks + options in `.yamllint.yml` :innocent: